### ISSUE TITLE
Download failed (on Steam Deck)

### ISSUE THREAD

**Issue by londiebrondie** [an-anime-team/an-anime-game-launcher#83]
![image](https://user-images.githubusercontent.com/125153020/222978005-dbae3e5b-5b4a-417e-afcb-90730ae4b35b.png)
[animefail.txt](https://github.com/an-anime-team/an-anime-game-launcher/files/10892561/animefail.txt)
Everything went well until this point. I've tried uninstalling and reinstalling with no success. I have 97gb free. Log is attached to post. 
Note: it did prompt me to install something like ".Net package runner" when I was going through setup. I did not mess with any of the setup options.

**Comment by toukoAMG** [comment]
is this the full log because it looks like 97% of it is not actually in the file you provided, since it says ```4771170B                                      1001,41        3%``` at line 25

**Comment by londiebrondie** [comment]
> is this the full log because it looks like 97% of it is not actually in the file you provided, since it says `4771170B 1001,41 3%` at line 25

I clicked the details button and it presented me with what I assumed to be a log, and I saved that and sent it here.

**Comment by LaJonquille** [comment]
Hello, have you find the bug beacause i have the same problem as you.
Thank for shrare your find.

**Comment by mschlachter** [comment]
I think this might be related to the amount of free space available. I'm getting a similar error using flatpak on Debian and the output on the console indicates that it requires 133.99GB (!) of free space for the initial install:
```sh
matthew@Persephone-debian:~$   2023-03-14T01:13:32.743665Z ERROR anime_game_launcher::ui::main: Downloading failed: No free space availale for specified path: "/home/matthew/.var/app/moe.launcher.an-anime-game-launcher/data/anime-game-launcher" (requires 133.99 GB, available 87.19 GB)
    at src/ui/main.rs:1263

  2023-03-14T01:13:33.078362Z ERROR anime_game_launcher::ui::main: Downloading failed: No free space availale for specified path: "/home/matthew/.var/app/moe.launcher.an-anime-game-launcher/data/anime-game-launcher" (requires 133.99 GB, available 87.19 GB)
    at src/ui/main.rs:1263

  2023-03-14T01:13:33.398967Z ERROR anime_game_launcher::ui::main: Downloading failed: No free space availale for specified path: "/home/matthew/.var/app/moe.launcher.an-anime-game-launcher/data/anime-game-launcher" (requires 133.99 GB, available 87.19 GB)
    at src/ui/main.rs:1263

  2023-03-14T01:13:33.726817Z ERROR anime_game_launcher::ui::main: Downloading failed: No free space availale for specified path: "/home/matthew/.var/app/moe.launcher.an-anime-game-launcher/data/anime-game-launcher" (requires 133.99 GB, available 87.19 GB)
    at src/ui/main.rs:1263
```

**Comment by londiebrondie** [comment]
> I think this might be related to the amount of free space available. I'm getting a similar error using flatpak on Debian and the output on the console indicates that it requires 133.99GB (!) of free space for the initial install:
> 
> ```shell
> 2023-03-14T01:13:32.743665Z ERROR anime_game_launcher::ui::main: Downloading failed: No free space availale for specified path: "/home/matthew/.var/app/moe.launcher.an-anime-game-launcher/data/anime-game-launcher" (requires 133.99 GB, available 87.19 GB)
>     at src/ui/main.rs:1263
> 
>   2023-03-14T01:13:33.078362Z ERROR anime_game_launcher::ui::main: Downloading failed: No free space availale for specified path: "/home/matthew/.var/app/moe.launcher.an-anime-game-launcher/data/anime-game-launcher" (requires 133.99 GB, available 87.19 GB)
>     at src/ui/main.rs:1263
> ```
> 
> Version 2.4.5 of the launcher installs the game without issue, and uses only 44.66GB for the initial install.

So should I try version 2.4.5 or should I try the latest version?
